python/lsst/afw/table/_syntax.py

```diff
@@ -202,3 +202,70 @@ def processArray(a):
         else:
             d[name] = processArray(self.get(schemaItem.key))
     return d
+
+def BaseCatalog_asAstropy(self, cls=None, copy=False, unviewable="copy"):
```

Comment:
It's "import astropy" so I don't think that people will think "asAstroPy".   I vote for asAstropy for consistency with the rest of our camelCaseCode.